src/_guides/language/language-tour.md

```diff
@@ -983,66 +983,62 @@ For more information about maps, see
 [Generics](#generics) and
 [Maps](/guides/libraries/library-tour#maps).
 
-### Runes
+<a id="characters"></a>
+### Runes and grapheme clusters
 
-In Dart, runes are the UTF-32 code points of a string.
+In Dart, [runes][] are the UTF-32 code points of a string.
+As of Dart 2.6, instead of using runes use the
+[characters package,][characters package]
+which has better support for user-perceived characters,
+also known as
+[Unicode (extended) grapheme clusters.][grapheme clusters]
 
 Unicode defines a unique numeric value for each letter, digit,
 and symbol used in all of the world's writing systems.
 Because a Dart string is a sequence of UTF-16 code units,
 expressing 32-bit Unicode values within a string requires
```

Comment:
"32-bit Unicode values" is vague and probably misleading. Just use "Unicode code points".

Code points are (so far) only 21-bit numbers.